benchmarks/bintrees.mjs

```diff
@@ -0,0 +1,226 @@
+import { group, bench, run } from 'mitata';
```

Comment:
I had too many other PRs outstanding so I didn’t post this yet, but I have a PR to replace benchmark-regressions with a workalike that wraps bench-node, which is maintained by a NodeJS core contributor. If I can get him to land one more PR I can clean up some junk in the output and then file that here, clearing the dead deps.